tests/typ/compiler/import.typ

```diff
@@ -78,16 +111,27 @@
 #import enum
 #let d = (e: enum)
 #import d.e
+#import d.e as renamed
 #import d.e: item
 
 #item(2)[a]
 
+---
+// Warning: 15-27 renaming imported name to its own name
```

Comment:
I think the warning message could somehow be nicer. Maybe `unnecessary rename to same name`?

Reply:
I like that, maybe `unnecessary import rename to same name`?